share/doc/wake/syntax/vscode/syntaxes/wake.tmLanguage.json

```diff
@@ -0,0 +1,155 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+	"scopeName": "source.wake",
+	"name": "wake",
+	"patterns": [
+		{
+			"include": "#expression"
+		}
+	],
+	"repository": {
+		"expression": {
+			"patterns": [
+				{ "include": "#comments" },
+				{ "include": "#parens" },
+				{ "include": "#keyword" },
+				{ "include": "#operator" },
+				{ "include": "#identifier" },
+				{ "include": "#literal" }
+			]
+		},
+		"parens": {
+			"name": "expression.group",
+			"begin": "\\(",
+			"end": "\\)",
+			"beginCaptures": { "0": { "name": "keyword.operator.wake" } },
+			"endCaptures": { "0": { "name": "keyword.operator.wake" } },
+			"patterns": [ { "include": "#expression" } ]
+		},
```

Comment:
Tried to make a real parser, but wake indentation is context sensitive => cannot be done with a simple parser.